addon/ExperimentProvider.js

```diff
@@ -1,60 +1,101 @@
-const seedrandom = require("common/vendor")("seedrandom");
-const simplePrefs = require("sdk/simple-prefs");
-const OVERRIDE_PREF = "experimentOverrides";
+const prefService = require("sdk/preferences/service");
+const {PrefsTarget} = require("sdk/preferences/event-target");
+const ss = require("sdk/simple-storage");
 
-exports.OVERRIDE_PREF = OVERRIDE_PREF;
 exports.ExperimentProvider = class ExperimentProvider {
-  constructor(clientID, experiments = require("../experiments.json"), rng) {
-    this._clientID = clientID;
+  constructor(experiments = require("../experiments.json"), rng) {
     this._experiments = experiments;
-    this._rng = rng || seedrandom(clientID);
+    this._rng = rng || Math.random;
```

Comment:
Thanks @ncloudioj, I'll remove those dependencies. @k88hudson yes, the prefs are persisted so we no longer a client to always have the same generated number